src/main/external-resources/UMS.conf

```diff
@@ -647,28 +647,55 @@ disable_transcoding =
 # Default: 6
 audio_channels =
 
+# AC-3 re-encoding audio bitrate (in Kbps) (e.g. 576, 640)
+# --------------------------------------------------------
+# Specify the bitrate for AC-3 transcoding; do not exceed the quality
+# supported by the AV receiver or amplifier.
+# Default: 640
+audio_bitrate =
+
+# Audio language priority
+# -----------------------
+# A comma-separated list of language codes that defines the priority for
+# MEncoder when choosing audio languages, where "und" stands for "undefined".
+# XXX Can be empty, which overrides the default value.
+# Default: a localized list (e.g. "eng,fre,jpn,ger,und").
+# audio_languages =
+
 # Use LPCM for audio
 # ------------------
 # Choose whether or not the Pulse Code Modulation (PCM) audio format should be
-# used instead of AC-3 with the MEncoder transcoding engine. Offers lossless
+# used instead of AC-3 with the MEncoder and FFmpeg transcoding engines. Offers lossless
 # transcoding quality but higher bitrate. Bandwidth requirements are around 1.5
 # Mbit/s for 2.0 and 4.6 Mbit/s for 5.1 audio. Not suitable for Wi-Fi
 # streaming.
+# This option remuxes LPCM, reencodes the other audio codecs to LPCM
+# (signed 16-bit little-endian) and overrided by each remux mode in the FFmpeg transcoding engine.
+# For this option to work, have to set MuxLPCMToMpeg  = true in renderer.conf.
 # Default: false
 audio_use_pcm =
 
-# Automatic audio resampling
-# --------------------------
-# Enable automatic audio resampling function used by FFmpegAudio for LPCM.
-# Default: true
-audio_resample =
+# Keep LPCM tracks ignoring profiles
+# ----------------
+# Remux LPCM instead of transcoding it in the FFmpeg video engine.
+# For this option to work, have to set MuxLPCMToMpeg  = true in renderer.conf.
+# Default: false
+audio_remux_lpcm =
 
 # Keep AC-3 tracks
 # ----------------
-# Remux AC-3 instead of transcoding it.
+# Remux AC-3 instead of transcoding it when using a profile sets ac3 in renderer.conf.
 # Default: true
 audio_remux_ac3 =
 
+# Keep AC-3 tracks, ignoring profiles
+# ----------------
+# Remux AC-3 instead of transcoding it in the FFmpeg video engine
+# even when using a profile not set ac3 in renderer.conf.
+# For this option to work, have to set ForceMuxAC3ToMpeg  = true in renderer.conf.
+# Default: false
+audio_force_remux_ac3 =
```

Comment:
Why is this setting needed? Isn't it better to make it automatic based on what the renderer supports?

Reply:
The behavior you refer is executed by `audio_remux_ac3 = true` in UMS.conf.
Its code is on `ac3Remux = params.aid.isAC3() && configuration.isAudioRemuxAC3() && params.mediaRenderer.isTranscodeToAC3();` in l. 1512 of FFMpegVideo.java.

The behavior of `audio_force_remux_ac3 = true` is to remux ac3 (`audio_remux_ac3 = true` is needed.) even when `TranscodeVideo =  MPEGTS-H264-AAC`. At least, it works well in my tests.

Reply:
I exclude Profile to wmv,  to be on the safe side,

Reply:
>  Some renderers will allow it while others won't

Yes. You are right. As a result, Both `ac3Remux = params.aid.isAC3()` and `audio_remux_ac3 = true` needed at the same time. This follows the behavior of remux DTS.

Reply:
Why should a user need to enable it? If they have a renderer that supports it, then it should be used, because it is better to mux than transcode.
I am thinking of the ease for users. When possible, good things should "just work"